charts/iaa-device-plugin/templates/iaa.yaml

```
@@ -7,14 +7,14 @@ deployments/operator/samples/deviceplugin_v1_iaadeviceplugin.yaml
 apiVersion: deviceplugin.intel.com/v1
 kind: IaaDevicePlugin
 metadata:
-  name: {{ .Values.iaa.metadata.name }}
-  annotations: {{ toYaml .Values.qat.metadata.annotations | nindent 4 }}
+  name: {{ .Values.metadata.name }}
+  annotations: {{ toYaml .Values.metadata.annotations | nindent 4 }}
 spec:
-  image: {{ .Values.iaa.spec.image }}
-  initImage: {{ .Values.iaa.spec.initImage }}
-  sharedDevNum: {{ .Values.iaa.spec.sharedDevNum }}
-  logLevel: {{ .Values.iaa.spec.logLevel }}
-  nodeSelector: {{- .Values.iaa.spec.nodeSelector | toYaml | nindent 4 }}
+  image: {{ .Values.spec.image }}
```

I think you also dropped `spec` so this becomes something like:

```suggestion
  image: "{{ .Values.image.hub }}/intel-iaa-plugin:{{ .Values.image.tag | default .Chart.AppVersion }}"
```